app/encryption.py

```diff
@@ -1,31 +1,7 @@
-from typing import Any, List, NewType, Optional, TypedDict, cast
+from typing import Any, List, cast
 
 from flask_bcrypt import check_password_hash, generate_password_hash
 from itsdangerous import URLSafeSerializer
-from typing_extensions import NotRequired  # type: ignore
-
-SignedNotification = NewType("SignedNotification", str)
-SignedNotifications = NewType("SignedNotifications", List[SignedNotification])
-
-
-class NotificationDictToSign(TypedDict):
```

Comment:
This was moved into the `types` module and became `PendingNotification` to represent the early lifecycle stage of the notification processing (when it hits the API and prior to be saved into the DB).